frontend/src/entities/configuration/configuration-reducer.ts

```diff
@@ -28,12 +29,13 @@ import { defaultConfiguration, IAppConfiguration } from '../../shared/model/app.
 const initialState = {
     appConfigList: [],
     appConfig: defaultConfiguration,
+    loadingAppConfig: false,
     failedToLoadConfig: false,
 };
 
 // Actions
 export const getConfiguration = (configScope: string) => {
-    return listConfigurations({ configScope, numVersions: 1});
+    return listConfigurations({ configScope, numVersions: 10});
```

Comment:
Don't do this. The intent of this function is to get a single config for normal users. Only admins are supposed to be able to get more than 1 config with the list configurations function.

Reply:
is there a downside to having all users have a list of 10 configurations in the store rather than just admins? I dont this this information is particularly sensitive since users had access to it anyways.

Reply:
@douglas1850 would have the best answer. My understanding is that general users shouldn't need anything other than the current. I don't think there is a strict reason why this would be a security issue, but limiting to one was preferred in the original design.

Reply:
The configs aren't sensitive information, but there's no reason to grab 10 when a typical user would only ever need 1. We're 10x'ing the number of reads from DDB for absolutely no reason. 

If you're not sure how to fetch configs, you can invoke the `listConfigurations` function directly like this:

```
dispatch(listConfigurations({configScope: 'global', numVersions: 10}));
```

Reply:
The problem with calling listConfigurations directly is that `appConfigList: data,` gets overwritten to only display 1 configuration. This is because we call getConfiguration in the routes component and then when I call listConfigurations in the history table component, it also gets set to num versions 1. I can probably pass an extra parameter to listConfigurations when calling it from getConfiguration so that we dont update appConfigList in that call

Reply:
We originally had separate reducers for `get` and `list` but switched to doing it this way to minimize repetitive code. 

It seems like this might be resolved by simply separating those out again. If `get` and `list` had its own reducer then we could make sure that the `get` calls weren't setting the appConfigList field, and that `list` calls don't set the appConfig field.